chain/chunks/src/lib.rs

```diff
@@ -397,13 +399,17 @@ pub struct ShardsManager {
     chunk_forwards_cache: SizedCache<ChunkHash, HashMap<u64, PartialEncodedChunkPart>>,
 
     seals_mgr: SealsManager,
+    // Useful to make tests deterministic and reproducible,
+    // while keeping the security of randomization of transactions in pool
```

Comment:
```suggestion
    /// Useful to make tests deterministic and reproducible,
    /// while keeping the security of randomization of transactions in pool
```
also it is worth explaining where exactly this `RngSeed` is used